tensorflow/core/api_def/base_api/api_def_StringLower.pbtxt

```diff
@@ -1,3 +1,11 @@
 op {
   graph_op_name: "StringLower"
+  summary: "Converts all uppercase characters into their respective lowercase replacements."
+  description: <<END
+Example:
+```
```

Comment:
No need for the ` ``` ` lines, just leave them empty, please.

Reply:
Done, was nor sure how description is going to be propagated from `pbtxt`